osmosis-1/osmosis.zone_assets.json

```diff
@@ -868,10 +868,6 @@
       "base_denom": "link-wei",
       "path": "transfer/channel-208/link-wei",
       "osmosis_verified": true,
```

Comment:
because it's no longer canonical, it needs:
```
"override_properties": {
  "logo_URIs": {
    "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/blob/master/osmosis/images/shib.axl.svg",
    "png": "https://raw.githubusercontent.com/cosmos/chain-registry/blob/master/osmosis/images/shib.axl.png"
  }
}
```

Reply:
same with the others, (shib.axl, arb.axl, pepe.axl, dot.pica, sol.wh)

Reply:
You can reference this for file names: https://github.com/cosmos/chain-registry/pull/5193/files